### PR TITLE
Flxcontrolhandler invert directions

### PR DESCRIPTION
**Pull request by winterismute**
Implements a simple way to invert movement of the character controlled by FlxControlHandler. Might be triggered by game effect f.e. to confuse the player.

### CODE REVIEW

File: flixel/addons/plugin/control/FlxControlHandler.hx [hunk 1]
```diff
@@ -188,6 +188,9 @@ class FlxControlHandler
 	private var _clockwiseKey:String;
 	private var _thrustKey:String;
 	private var _reverseKey:String;
+	// Signal movement inverted on hor/vert axis
+	private var invertMovementHorizontally(get, null):Bool;
```
Comment (Gama11) [inline flixel/addons/plugin/control/FlxControlHandler.hx]: `(get, null)` makes no sense for `private` properties.

**Comment by Gama11** [comment]
I think instead of making these properties and having a `invertMovement()` function, these should be regular, public variables. Perhaps named `flipX` and `flipY` to match `FlxSprite` API?

**Comment by Gama11** [comment]
"invert" may be more natural for movement than "flip" though, not sure...

**Comment by winterismute** [comment]
@Gama11 well, I see that in the whole flixel there are different API conventions for different classes... But yes, maybe I should switch to having just "invertX" and "invertY" as public bools, in the end setters do not do any additional work... What do you think?

**Comment by Gama11** [comment]
Sounds good.

**Comment by winterismute** [comment]
@Gama11 This might be done now.

**Comment by Gama11** [comment]
Thanks for the heads-up, there's no GitHub notifications for new commits on pull requests.

**Pull request merged**